lib/experimental/block-bindings/index.php

```diff
@@ -7,13 +7,12 @@
 
 require_once __DIR__ . '/class-wp-block-bindings.php';
 require_once __DIR__ . '/block-bindings.php';
+require_once __DIR__ . '/sources/post-meta.php';
```

Comment:
To move the post meta source out of the experiment we need to finalize changes planned in https://github.com/WordPress/gutenberg/pull/58105 first.